### ISSUE TITLE
Finding the issue with enum34

### ISSUE THREAD

**Issue by GTimothee** [big-data-lab-team/sam#21]
I had an issue with the enum package when using sam for the first time. Valérie suggested I reproduce the error to see what the issue was and potentially adding a note to the README.

Precision: The issue was that sam needed enum install, so I installed enum but it did not work because enum34 was expected instead of enum. 

I will try to reproduce the issue when I have time.

**Comment by glatard** [comment]
sounds like the module should be added as a dependency in `setup.py`.

**Comment by ValHayot** [comment]
Agreed. I was going to add it, but I don't have the module `enum` in python3 (python2.7 only) and I didn't install it in Travis, and somehow that passes too. Just wanted to investigate further before forcing users to install a module.
